benchmark/benchmarks.jl

```diff
@@ -118,28 +138,30 @@ SUITE["graph"]["add_union"]["agent_pos"] =
 SUITE["graph"]["add_union"]["agent_single"] =
     @benchmarkable add_agent_single!($graph_agent, $graph_union_model)
 
-graph_model = ABM(GraphAgent, GraphSpace(complete_digraph(1000)))
-for _ in 1:800
+graph_model = ABM(GraphAgent, GraphSpace(complete_digraph(2000)))
+for _ in 1:1800
     add_agent!(graph_model, 6.5, false)
 end
 a = random_agent(graph_model)
+pos = rand(1:2000)
```

Comment:
but this still adds agents to random positions. I would claim that we should make a complete digraph with 100 nodes and add 10 agents in each node for example.

Reply:
Then it would not matter from which `pos` you search, because all will have the same **amount of numerical operations**